res/database/Default/user/stukov_ads.xml

```diff
@@ -640,6 +701,7 @@
 			<description>
 				<de>Nous translate alles!</de>
 				<en>Nous translate alles!</en>
+				<pl>Maple-Translate tłumaczy wszystko!</pl>
```

Comment:
so the translation itself is the joke here. `;)`
maybe leave it like in the other versions or just translate "alles" to Polish?

Reply:
Acha. Clever!!! :)